.env

```diff
@@ -0,0 +1,6 @@
+BROWSER=chrome
```

Comment:
@makarandp0 Is this file necessary to be committed, since you're adding it to `.gitignore`?

Reply:
This file is optional - but  helps with local runs, so I am committing it as a recommended template.  I added it gitignore so that git won't track any local changes to this file.

Reply:
@makarandp0 Then I think we should rename it to `.env.template`.